src/docs/command-line-deployment.md

```diff
@@ -1,35 +1,51 @@
 # Command-Line Deployment
 
+The steps in this article assume the following pre-requisites for command-line deployments:
+
+* Follow the Mission LZ [Getting Started](https://github.com/Azure/missionlz/blob/main/src/docs/getting-started.md#pre-requisites) steps.
+* **(Highly recommend)** Use the [the Mission LZ `.devcontainer`](https://github.com/Azure/missionlz/blob/main/src/docs/getting-started.md#use-the-development-container-for-command-line-deployments) provided in the Mission LZ project and perform the deployment steps below within this context. This container image provides a controlled environment that includes all the pre-requisite tools for Mission LZ deployments and should lead to an overall better user experience.
+
+  > As an alternative, it is possible to deploy Mission LZ via BASH running from the local workstation, but requires the following additional requirements:
+  >
+  > * An Azure Subscription where you have ['Owner' RBAC permissions](<https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/>)
+  > * The current version of Azure CLI (try `az cli -v` or see <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli/>)
+  > * Terraform CLI version > v0.13.4 (try `terraform -v` or see <https://learn.hashicorp.com/tutorials/terraform/install-cli/>)
+
 ## Step-by-step
 
+1. Follow the [steps to open the `.devcontainer`](../../.devcontainer/README.md) as the recommended option (or start a local BASH shell with the additional requirements installed as the alternate option)
+
+   > `vscode@missionlz-dev:/workspaces/missionlz$` is the root working directory for the BASH shell in the `.devcontainer`
+
 1. Log in using the Azure CLI
 
     ```BASH
     az login
     ```
 
-1. [Quickstart](#Quickstart)
-1. [Setup Mission LZ Resources](#Setup-Mission-LZ-Resources)
-1. [Set Terraform Configuration Variables](#Set-Terraform-Configuration-Variables)
-1. [Deploy Terraform Configuration](#Deploy-Terraform-Configuration)
-1. [Clean up Mission LZ Resources](#Clean-up-Mission-LZ-Resources)
+   > *(Optional)* If you needed to deploy into another cloud such as Azure Government, set the cloud name before logging in:
+
+     ```BASH
+     az cloud set -n AzureUSGovernment
+     az login
+     ```
 
-### Quickstart
+1. [Quickstart](#quickstart)
+1. [Setup Mission LZ Resources](#setup-mission-lz-resources)
```

Comment:
Suggestion: how about separating the link to the Quickstart here so that it's apparent that someone gets a full deployment from the quickstart, and then they can also customize the deployment following the guidance in the other sections. I think the way it is makes the Quickstart look like step one of a multi-step process rather than a complete deployment. For example:

(replacing line 33): You can execute a complete deployment with default configuration settings using the [Quickstart](#quickstart) guidance. If you want to customize the deployment, follow the additional guidance below:
1. [Setup Mission LZ Resources](#setup-mission-lz-resources)
1. ...
1. ...